tests/skylab/smartseq2_single_nucleus/pr/ValidateSmartSeq2SingleNucleus.wdl

```diff
@@ -0,0 +1,42 @@
+version 1.0
+
+task ValidateSnSmartSeq2 {
+    input {
+      File exon_intron_counts_hash
+      String truth_exon_intron_counts_hash
+      File loom_output
+      File truth_loom
+
+      Int disk_size = ceil(size(loom_output,"GiB") + size(truth_loom, "GiB") + 10)
+    }
+
+  command <<<
+
+    # catch intermittent failures
+    set -eo pipefail
+
+    #compare looms
+    python3 /tools/loomCompare.py --truth-loom ~{truth_loom} --check-loom ~{loom_output} --delta-cutoff 10
+
+    # calculate hashes; awk is used to extract the hash from the md5sum output that contains both
+    # a hash and the filename that was passed. We parse the first 7 columns because a bug in RSEM
+    # makes later columns non-deterministic.
+
+    #commenting this line out until we can pull a file from the snSS2 outputs
+    exon_intron_counts_hash=$(awk 'NR>2' "~{exon_intron_counts_hash}" | md5sum | awk '{print $1}')
+
+    if [ "$exon_intron_counts_hash" != "~{truth_exon_intron_counts_hash}" ]; then
+        echo "exon_intron_counts_hash "$exon_intron_counts_hash" did not match expected hash "~{truth_exon_intron_counts_hash}""
+        fail=true
```

Comment:
Why not just exit 1; here?  Instead of below?

Reply:
Fixed this, thanks!